tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -332,6 +335,18 @@
         </convert>
     </target>
 
+    <target name="copy-cldrtestdata" depends="init-args">
+        <copy todir="${testDataDir}/cldr-testData">
+            <fileset dir="${cldrDir}/common/testData">
+                <include name="units/**"/>
+            </fileset>
+        </copy>
+    </target>
+
+    <target name="clean-cldrtestdata" depends="init-args">
+        <delete dir="${testDataDir}/cldr-testData"/>
+    </target>
+
```

Comment:
Why doesn't "clean" depend on clean-cldrtestdata ? I think I'd expect if I run "clean" then it's back to the same state as when I download the repository.

Reply:
Done and tested.